### ISSUE TITLE
No Spell Check in chrome

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#573]
```
What's the problem?
Google chrome spell check does not appear to be using




What browser are you using?

Google Chrome	17.0.963.56 (Official Build 121963) m
OS	Windows
WebKit	535.11 (@107413)
JavaScript	V8 3.7.12.22
Flash	11,1,102,62
User Agent	Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/535.11 (KHTML, like 
Gecko) Chrome/17.0.963.56 Safari/535.11




What version of ScribeFire are you running? 4.1
```

-----
Original issue reported on code.google.com by `ZanePe...@gmail.com` on 17 Feb 2012 at 5:07

**Comment by GoogleCodeExporter** [comment]
```
The same problem is in Opera 12.02 on Windows 7 and Ubuntu 12.04.
```

-----
Original comment by `Bartosz....@gmail.com` on 9 Sep 2012 at 3:07

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:08
* Changed state: **Fixed**
